solutionbox/code_free_ml/mltoolbox/code_free_ml/analyze.py

```diff
@@ -158,6 +158,76 @@ def parse_arguments(argv):
   return args
 
 
+def make_analysis_plan(schema, features):
```

Comment:
The one thing I like about the plan concept is that we don't have to map each individual transform to an analysis step in two places (run_local_analysis and run_cloud_analysis), now it is done in one place (make_analysis_plan)